src/content/quickstart/_index.en.md

```diff
@@ -45,6 +45,32 @@ An example of three clusters configured to use with Submariner would look like t
 | east         | OnPrem   | 10.98.0.0/16 | 10.99.0.0/16 | 192.168.1.0/24     |
 
 
+
+## Support Matrix
+
+Submariner is designed to be cloud provider agnostic, and should run in any standard Kubernetes cluster. Submariner has been tested with and known to be working properly with the following cloud environments:
+
+{{% notice tip %}}
+Are you using Submariner in an environment which is not described here? Please [let us know](../contributing/website) so that we can update this document. 
+{{% /notice %}}
+
+### Cloud Providers
+
+* AWS
+* VMware vSphere
+* OpenStack
+* GCP
+
+### Kubernetes CNI Plugins
+
+Presently, Submariner has been tested with the following CNI Plugins that leverage kube-proxy with iptables mode:
+
+* [openshift-sdn](https://github.com/openshift/sdn)
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
```

Comment:
nit: Do we want to capture the open issue we have with Canal?
i.e., https://github.com/submariner-io/submariner/issues/407

Reply:
I wanted to take a "white-list" approach where we mention what known to be working. Otherwise, we could have mentioned that OVN is not supported today, etc. Note that canal != calico (my understanding is that Rancher uses canal today in their offering, @Oats87 can you confirm?)